src/storage/src/hummock/local_version_manager.rs

```diff
@@ -213,6 +217,23 @@ impl LocalVersionManager {
         local_version_manager
     }
 
+    pub async fn for_test(
```

Comment:
We should add a `#[cfg(test, feature = "test")]` to avoid compile it in release mode.